src/states/pdf.py

```diff
@@ -0,0 +1,18 @@
+import random
+import string
+from typing import Literal, Optional
+
+from pydantic import BaseModel
+
+
+class PDFUploaderState(BaseModel):
+	name: str = "uploader"
+	key: str = ""
+	type: str = "pdf"
+	is_file_uploaded: bool = False
+	file: Optional[bytes] = None
```

Comment:
In python > 3.10 you don't need to use Optional from typing. 
Better syntax for me is:
```file: bytes | None = None```